### PR TITLE
Create a component to handle floating label and slider value 

### PR DESCRIPTION
**Pull request by brunobc182**
# What
Fix floating label on inputs page and slider value on controls & toggles page.

# Why
The floating labels don't work properly when we navigate at docs. They only work when we reload the entire page. And slider value doesn't work the way it should changed it.

# How
- Create a component on `input.js` to call at `input.mdx` file.
- Create a component on `slider.js` to call at `controls-toggle.mdx` file.

### CODE REVIEW

**Comment by brunobc182** [comment]
[Investigar problema de carregamento de JS na documentação](https://trello.com/c/lB9WtW96/380-investigar-problema-de-carregamento-de-js-na-documenta%C3%A7%C3%A3o)

File: docs/inputs.mdx [hunk 1]
```diff
@@ -2,9 +2,10 @@
 name: Inputs
 ---
 
+import {Fragment} from 'react'
```
Comment (larimaza) [inline docs/inputs.mdx]: Just 💅 hahaha
```suggestion
import { Fragment } from 'react'
```

File: docs/js/slider.js [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
       const sliderValue = slider.value;
 
       slider.style.setProperty("--slider-percentage", `${sliderValue}%`);
       counter.innerText = `${sliderValue} coins`;
     });
-  })
+  });
+};
+
+const HandleSliderValue = () => {
```
Comment (kuroski) [inline docs/js/slider.js]: Maybe you could change to do that

``` javascript
const HandleSliderValue = changeSliderValue;
```

Or just

``` javascript
export default changeSliderValue
```

Comment (brunobc182) [reply docs/js/slider.js]: I did this because someone could just import the function and we need the component way to put on `mdx` file.

**Pull request merged**